src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1510,4 +1510,25 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="PreferDictionaryContainsKeyCodeFixTitle" xml:space="preserve">
+    <value>Use 'ContainsKey(TKey)'</value>
+  </data>
+  <data name="PreferDictionaryContainsKeyDescription" xml:space="preserve">
+    <value>'{0}.ContainsKey(TKey)' is usually O(1), while '{0}.Keys.Contains(TKey)' may be O(n) in some cases. Additionally, many dictionary implementations lazily initialize the Keys collection to cut back on allocations.</value>
```

Comment:
You cannot parameterize the description string. Please remove all the `{0}` usages.

Reply:
Good to know. I'll remove them.